binder/appendix.txt

```diff
@@ -8,12 +15,8 @@ RUN CONDA_DIR=/srv/conda && \
     echo "export RADARENV=$RADARENV" >> ~/.profile && \
     echo "export CONDA_PREFIX=$CONDA_PREFIX" >> ~/.profile && \
     ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/baltrad/install_baltrad.sh && \
-    apt update && \
-    apt-get -y install sudo && \
-    sudo apt-get -y update && \
-    sudo -s && \
-    sudo ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/lrose/build_lrose.sh && \
     ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/nbstripout_notebooks.sh && \
```

Comment:
Can we fix this typo here too?

Reply:
Oh, has it been there from the beginning?

Reply:
I think so...